src/main/java/core/basesyntax/RandomValue.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomValue extends EnumColors {
+    public static Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return EnumColors.Color.values()[index];
+    }
+
+    public static int getRandomNumber() {
+        Random random = new Random();
+        int bound = 100;
+        return random.nextInt(bound);
+    }
+}
```

Comment:
You don't need this class. Move the functionality to ColorSupplier/Lottery
Create one random object per each class instance, not per each method invocation (make it a class field)
100 should be a constant